### PR TITLE
[NOTICKET] add configurations for rabbitmq and elasticsearch. Enable configurable additional php extensions.

### PR DESCRIPTION
**Pull request by jochenmanz**

### CODE REVIEW

File: devenv.nix [hunk 1]
```diff
@@ -149,11 +152,32 @@ in {
       '';
     };
 
+    additionalPhpExtensions = lib.mkOption {
+      type = lib.types.listOf lib.types.str;
+      description = "Additional php extensions";
```
Comment (kleinmann) [inline devenv.nix:157]:
```suggestion
      description = "Additional PHP extensions";
```

File: devenv.nix [hunk 2]
```diff
@@ -149,11 +152,32 @@ in {
...
     additionalCaddyVhostConfig = lib.mkOption {
       type = lib.types.str;
       default = "";
       description = "Additional caddy vhost configuration";
     };
+
+    enableElasticSearch = lib.mkOption {
+      type = lib.types.bool;
+      default = false;
+      description = "Enables ElasticSearch";
```
Comment (kleinmann) [inline devenv.nix:173]:
```suggestion
      description = "Enable Elasticsearch";
```

File: devenv.nix [hunk 3]
```diff
@@ -149,11 +152,32 @@ in {
...
+    };
+
+    enableRabbitMq = lib.mkOption {
+      type = lib.types.bool;
+      default = false;
+      description = "Enables RabbitMq";
```
Comment (kleinmann) [inline devenv.nix:179]:
```suggestion
      description = "Enable RabbitMQ";
```

File: devenv.nix [hunk 4]
```diff
@@ -149,11 +152,32 @@ in {
...
+      ];
+    };
+
     additionalCaddyVhostConfig = lib.mkOption {
       type = lib.types.str;
       default = "";
       description = "Additional caddy vhost configuration";
     };
+
+    enableElasticSearch = lib.mkOption {
```
Comment (kleinmann) [inline devenv.nix:170]:
```suggestion
    enableElasticsearch = lib.mkOption {
```

**Review by kleinmann — CHANGES_REQUESTED** [review]
Just a bit of naming polish, looks good otherwise 🙂

**Comment by Zwaen91** [comment]
I would suggest adding the new possible configurations to the README file.

File: devenv.nix (hunk 3 above)
Comment (jochenmanz) [reply devenv.nix]: Done :)

File: devenv.nix (hunk 2 above)
Comment (jochenmanz) [reply devenv.nix]: Done :)

File: devenv.nix (hunk 1 above)
Comment (jochenmanz) [reply devenv.nix:157]: Done :)

**Comment by jochenmanz** [comment]
> I would suggest adding the new possible configurations to the README file.

Ive added more documentation to our readme.md

File: devenv.nix (hunk 4 above)
Comment (jochenmanz) [reply devenv.nix]: Done :)

**Review by kleinmann — APPROVED** [review]

**Pull request merged**